### PR TITLE
OSSMDOC-553: merge distributed tracing attributes files. 

### PR DESCRIPTION
**Pull request by JStickler**
xref:#44070 for service mesh attributes
[OSSMDOC-218](https://issues.redhat.com/browse/OSSMDOC-218) - Update the attributes to avoid confusion
[OSSMDOC-553](https://issues.redhat.com/browse/OSSMDOC-553) - Merge the Jaeger and distributed tracing attributes files into the OpenShift common-attributes file.  
* Updates all attributes to unique names.
* Moves attributes from independent files to common-attributes
* Removes two independent attribute files.
* Updates attribute references in all Jaeger and distributed tracing assemblies.

Previews
https://deploy-preview-44067--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html
https://deploy-preview-44067--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html
https://deploy-preview-44067--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_arch/distr-tracing-architecture.html
https://deploy-preview-44067--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-installing.html
https://deploy-preview-44067--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-deploying-jaeger.html
https://deploy-preview-44067--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-deploying-otel.html
https://deploy-preview-44067--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-updating.html
https://deploy-preview-44067--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_install/distr-tracing-removing.html

### CODE REVIEW

**Comment by JStickler** [comment]
@abrennan89 , FYI, I'm not sure if these files that use the ProductName variable want to be updated to SMProductName?  Or JaegerName (I know you used to have a file in the Jaeger Config directory).
            ▪ machine_management/creating-infrastructure-machinesets.adoc
            ▪ modules/serverless-ossm-enabling-serving-metrics.adoc
            ▪ modules/serverless-ossm-setup-with-kourier.adoc
            ▪ modules/serverless-ossm-setup.adoc
            ▪ post_installation_configuration/cluster-tasks.adoc
            ▪ post_installation_configuration/network-configuration.adoc
            ▪ serverless/admin_guide/serverless-ossm-setup.adoc

**Comment by rh-max** [comment]
@abrennan89 I found no instances of these attributes:
* `_attributes/distr-tracing-document-attributes.adoc`
* `_attributes/jaeger-document-attributes.adoc`

in `serverless/`, however I found four instances of:

* `_attributes/ossm-document-attributes.adoc`

and deleted them. https://github.com/openshift/openshift-docs/pull/44080

Thanks.

**Comment by JStickler** [comment]
@rh-max  FYI, you'll probably also want to search the attributes themselves to make sure deleting the reference didn't break anything for Serverless.   `ProductName` in our attributes files has changed to be `SMProductName` (for service mesh), or DTProductName (for distributed tracing).

**Review by kalexand-rh — COMMENTED** [review]
I have one comment, but this is looking great!

File: _attributes/common-attributes.adoc [hunk 1]
```diff
@@ -71,8 +71,27 @@ endif::[]
 ifdef::openshift-origin[]
 :VirtProductName: OKD Virtualization
 endif::[]
+//distributed tracing
+:DTProductName: Red Hat OpenShift distributed tracing
+:DTShortName: distributed tracing
+:DTProductVersion: 2.3
+:JaegerName: Red Hat OpenShift distributed tracing platform
```
[the remark this answers is not in the document]
Comment (JStickler) [reply _attributes/common-attributes.adoc:78]: OpenShift Jaeger rebranded to Red Hat distributed tracing (which has two components, Red Hat OpenShift distributed tracing platform and Red Hat OpenShift distributed tracing data collection).  So not confusing at all, I know.  I went through the rebranding PRs with a fine tooth comb, so hopefully I've got everything set correctly.

[the remark this answers is not in the document]
Comment (kalexand-rh) [reply _attributes/common-attributes.adoc:78]: Nope, not confusing at all! This is why I didn't want to try to do this myself!

**Pull request merged**